doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -14,7 +14,7 @@ The following is a list of the parameters in the ``config_defaults.yaml`` file.
 
 .. _user:
 
-USER Configuration Parameters
+USER-related Configuration Parameters
 =================================
```

Comment:
The Doc Tests GitHub Action is failing due to the title underline being too short.  Please increase the size of the underline as appropriate.

Reply:
I fixed this, thank you for pointing that out